python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -566,8 +632,28 @@ def add_edge_data(
 
         Examples
         --------
-        >>>
+        >>> import cugraph
+        >>> import cudf
+        >>> dataset2 = {
+        ...     "simple": [
+        ...         ["src", "dst", "some_property"],
+        ...         [
+        ...             (99, 22, "a"),
+        ...             (98, 34, "b"),
+        ...             (97, 56, "c"),
+        ...             (96, 88, "d"),
+        ...         ],
+        ...     ],
+        ... }
+        >>> from cugraph.experimental import PropertyGraph
+        >>> dataframe_type = cudf.DataFrame
+        >>> simple = dataset2["simple"]
+        >>> pG = PropertyGraph()
+        >>> df = cudf.DataFrame(columns=simple[0], data=simple[1])
+        >>> pG.add_edge_data(df, vertex_col_names=("src", "dst"))
+        >>> pG.get_edge_data()
```

Comment:
Can you also update this example using the suggestions for `get_num_vertices()` above?  You should also make sure the example runs and include the return value of the `get_edge_data()` call, since I think doctest will use that.

Reply:
Done